charts/redpanda/values.go

```diff
@@ -142,6 +197,34 @@ type Auth struct {
 	SASL *SASLAuth `json:"sasl" jsonschema:"required"`
 }
 
+func (a *Auth) IsSASLEnabled() bool {
+	if a.SASL == nil {
+		return false
+	}
+
+	return a.SASL.Enabled
+}
+
+func (a *Auth) Translate(isSASLEnabled bool) map[string]any {
+	result := map[string]any{}
+	if a.SASL == nil || !isSASLEnabled {
```

Comment:
nit: Should this just be `a.IsSASLEnabled`?

Reply:
You are right